node/src/accountant/payment_adjuster/mod.rs

```diff
@@ -0,0 +1,2340 @@
+
+    #[test]
+    fn defaulted_calculators_react_on_correct_params() {
+        // When adding a test case for a new calculator, you need to make an array of inputs. Don't
+        // create brand-new accounts but clone the provided nominal accounts and modify them
+        // accordingly. Modify only those parameters that affect your calculator.
+        // It's recommended to orientate the modifications rather positively (additions), because
+        // there is a smaller chance you would run into some limit
+        let input_matrix: InputMatrixConfigurator =
+            |(nominal_account_1, nominal_account_2, _now)| {
+                vec![
+                    // First test case: BalanceCalculator
+                    {
+                        let mut account_1 = nominal_account_1;
+                        account_1.bare_account.balance_wei += 123_456_789;
+                        let mut account_2 = nominal_account_2;
+                        account_2.bare_account.balance_wei += 999_999_999;
+                        [(account_1, 8000001876543209), (account_2, 8000000999999999)]
+                    },
+                ]
+            };
+        // This is the value that is computed if the account stays unmodified. Same for both nominal
+        // accounts.
+        let current_nominal_weight = 8000000000000000;
+
+        test_calculators_reactivity(input_matrix, current_nominal_weight)
+    }
+
+    #[derive(Clone, Copy)]
+    struct TemplateComputedWeight {
+        common_weight: u128,
+    }
+
+    struct ExpectedWeightWithWallet {
+        wallet: Wallet,
+        weight: u128,
+    }
+
+    fn test_calculators_reactivity(
+        input_matrix_configurator: InputMatrixConfigurator,
+        nominal_weight: u128,
+    ) {
+        let calculators_count = PaymentAdjusterReal::default().calculators.len();
+        let now = SystemTime::now();
+        let cw_service_fee_balance_minor = gwei_to_wei::<u128, u64>(1_000_000);
+        let (template_accounts, template_computed_weight) =
+            prepare_nominal_data_before_loading_actual_test_input(
+                now,
+                cw_service_fee_balance_minor,
+            );
+        assert_eq!(template_computed_weight.common_weight, nominal_weight);
+        let mut template_accounts = template_accounts.to_vec();
+        let mut pop_account = || template_accounts.remove(0);
+        let nominal_account_1 = pop_account();
+        let nominal_account_2 = pop_account();
+        let input_matrix = input_matrix_configurator((nominal_account_1, nominal_account_2, now));
+        assert_eq!(
+            input_matrix.len(),
+            calculators_count,
+            "If you've recently added in a new calculator, you should add in its new test case to \
+            this test. See the input matrix, it is the place where you should use the two accounts \
+            you can clone. Make sure you modify only those parameters processed by your new calculator "
```

Comment:
Since you're asserting on equality this statement will make sense when the length of matrix is less than the calculators_count but if it's greater, the statement will loose it meaning.

Either change the assertion or the message according to your convenience.